rest/route/github_test.go

```diff
@@ -23,10 +23,12 @@ import (
 	"github.com/mongodb/amboy"
 	"github.com/mongodb/grip"
 	"github.com/stretchr/testify/suite"
+	"go.mongodb.org/mongo-driver/bson"
 )
 
 type GithubWebhookRouteSuite struct {
-	sc                     *data.MockConnector
+	ss                     *data.MockConnector
```

Comment:
I think the c here typically means connector; could we have ss be mock or something so it's clear? (I know we intend to get rid of this ASAP but just in case we make the mock more of a subclass of the DB connector in the future)